pkg/api/validation/validation.go

```diff
@@ -1614,6 +1615,17 @@ func ValidateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 		if mountpoints.Has(mnt.MountPath) {
```

Comment:
...or starting with `./`, which shouldn't be allowed.  I noticed this because I was wondering what kind of test case we'd have to construct to test an error being returned from `filepath.Rel`.  I wonder if we want to add some additional validations here.